pkg/config/config.go

```diff
@@ -80,6 +80,7 @@ var (
 	BindAddressKey               = "BIND_ADDRESS"
 	CPUArchOverride              = getConfig("CPU_ARCH_OVERRIDE", "")
 	MaxLookupRetry               = getIntConfig("MAX_LOOKUP_RETRY", defaultMaxLookupRetry)
+	BPFSampleRate                = getIntConfig("BPF_SAMPLE_RATE", 1000)
```

Comment:
Shouldn't we use smaller values?

Reply:
sure, let's have some stats first so users know what to choose and start from small values as default.